lib/FFI/Platypus/Lang/Win32.pm

```diff
@@ -11,8 +11,31 @@ use Config;
 =head1 SYNOPSIS
 
  use FFI::Platypus;
- my $ffi = FFI::Platypus->new( api => 1 );
- $ffi->lang('Win32');
+ use FFI::Platypus::Memory;  # For malloc and free.
+
+
+ use constant MB_OK => 0;
+
+ my $ffi_user32 = FFI::Platypus->new( api => 1 );
+ $ffi_user32->lang('Win32');
+ $ffi_user32->find_lib( lib => 'user32' );
+
+ $ffi_user32->attach([MessageBoxW => 'MessageBox'] => ['HWND', 'LPCWSTR', 'LPCWSTR', 'UINT'] => 'int');
+
+ MessageBox(undef, "I ❤️ Platypus", "Confession", MB_OK);
```

Comment:
To get this to work I had to add `MB_DEFAULT_DESKTOP_ONLY` (not sure why) and `use utf8` (which makes sense tbh).